draft-ietf-tls-tls13.md

```diff
@@ -3131,7 +3185,8 @@ aspect of the certificate chain was unacceptable (e.g., it was not signed by a
 known, trusted CA), the server MAY at its discretion either continue the
 handshake (considering the client unauthenticated) or abort the handshake.
 
-Any endpoint receiving any certificate signed using any signature algorithm
+Any endpoint receiving any certificate that must be validated and is
+signed using any signature algorithm
```

Comment:
Yes, they usually would not need to be sent, but they could be sent.
A connection could also be using opportunistic encryption and not plan on validating anything at all.
It seems silly to impose a requirement that "even if you're not going to do any other validation of any certificates, you still need to check for MD5 and reject it".
But, if I want to keep arguing this, I should take it to the list, and I'm not sure that I feel strongly enough to do so.